docs/installation/index.md

```diff
@@ -144,3 +136,36 @@ katello-installer <options>
 
 Katello provides a git repository designed to streamline setup by setting up all the proper repositories. Katello deploy provides the ability to deploy a virtual machine instance via Vagrant or direct deployment on an already provisioned machine. For details on how to install using katello-deploy, please see the [README](https://github.com/Katello/katello-deploy/blob/master/README.md).
 
+<script src="/assets/js/jquery.min.js"></script>
```

Comment:
This should already be being included - https://github.com/Katello/katello.org/blob/master/_layouts/base.html#L24